backend/.env.example

```diff
@@ -15,6 +15,13 @@ ADMIN_EMAIL=
 #FULL_PARSE_SCHEDULE=
 #UPDATE_PARSE_SCHEDULE=
 
+# --- google api ---
+SERVICE_ACCOUNT_JSON=path_to_json
+GOOGLE_PROJECT_ID=PROJECT_ID
```

Comment:
is this not included in the json?

Reply:
i see, this is for a different place...

Reply:
this still feels a little bit redundant. cant we just extract thath from the json?